applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_residual_based_newton_raphson_strategy.hpp

```diff
@@ -573,7 +573,7 @@ class MPMResidualBasedNewtonRaphsonStrategy
             }
             else
             {
-                KRATOS_INFO("MPM_Strategy") << "WARNING: no free DOFs!! " << std::endl;
+                KRATOS_WARNING("MPM_Strategy") << "WARNING: no free DOFs!! " << std::endl;
```

Comment:
I think this is an error in the core strategies ...?

Reply:
I am not sure about this...

Reply:
ok then it is in the builderandsolver

Reply:
Yes, it is there :+1: